R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -177,6 +177,12 @@ test_that("add and get file to be downloaded with Spark job on every node", {
   spark.addFile(path)
   download_path <- spark.getSparkFiles(filename)
   expect_equal(readLines(download_path), words)
+
+  # Test spark.getSparkFiles works well on executors.
+  seq <- seq(from = 1, to = 10, length.out = 5)
+  f <- function(seq) { spark.getSparkFiles(filename) }
+  spark.lapply(seq, f)
```

Comment:
I added check that filename getting from ```spark.getSparkFiles``` should be consistent with the original filename. We can't check the path, since the path get from driver and executors may different.